packages/flex-dev-utils/src/env.ts

```diff
@@ -0,0 +1,59 @@
+export type Environment = 'production' | 'development';
+export type Lifecycle = 'build' | 'prebuild' | 'deploy' | 'predeploy';
+
+/**
+ * Helper method to test whether env variable is defined
+ * @param key the env to lookup
+ * @return whether the key is set
+ */
+const isDefined = (key: string | undefined) => typeof key === 'string' && key !== '';
+
+export const isTerminalPersisted = () => process.env.PERSIST_TERMINAL === 'true';
+export const isCI = () => process.env.CI === 'true';
+export const isDebug = () => process.env.DEBUG === 'true';
+export const isVerbose = () => process.env.TRACE === 'true';
+export const getAccountSid = () => process.env.TWILIO_ACCOUNT_SID;
+export const getAuthToken = () => process.env.TWILIO_AUTH_TOKEN;
+export const getRealm = () => process.env.REALM;
+export const hasHost = () => isDefined(process.env.HOST);
+export const getHost = () => process.env.HOST;
+export const setHost = (host: string) => process.env.HOST = host;
+export const hasPort = () => isDefined(process.env.PORT);
+export const getPort = () => Number(process.env.PORT);
+export const setPort = (port: number) => process.env.PORT = String(port);
+export const getNodeEnv = () => process.env.NODE_ENV;
+export const setNodeEnv = (_env: Environment) => process.env.NODE_ENV = _env;
+export const getBabelEnv = () => process.env.BABEL_ENV;
+export const setBabelEnv = (_env: Environment) => process.env.BABEL_ENV = _env;
+export const getLifecycle = () => process.env.npm_lifecycle_event;
```

Comment:
nit: `NPM_LIFECYCLE_EVENT` to be consistent with the rest

Reply:
That's an env set by npm automatically, not me.